R/tables.R

```diff
@@ -0,0 +1,63 @@
+
+## If you use filter parameter you need to add styles:
+SEMANTIC_DT_STYLE <- shiny::tags$style("
```

Comment:
What about moving this to inst folder and attaching css file inside semanticPage?

Reply:
I'm just afraid that with nesting more and more js/css files this becomes too heavy for loading. This style tags are added only when called.

Reply:
At the next step I think we'll minimize and bundle all the css and js scripts so that it's lightweight. We already have an example in semantic.dashboard.
The next argument for moving this to a separate file is not repeating the styling if we want to render multiple tables. This way it's even more convenient to move it to css file.

Reply:
Okay, I'm convinced ;) moved